src/widgets/school-menu/ui/school-item/school-item.tsx

```diff
@@ -18,7 +19,13 @@ export const SchoolItem = ({ item, color }: SchoolItemProps) => {
   const descriptionContent = (
     <>
       <span className={color}>{item.title}</span>
-      <small>{descriptionText}</small>
+      {courseDate
+        ? (
+            <DateStart className="description" date={courseDate}></DateStart>
```

Comment:
```<DateStart className="description" date={courseDate} />```

offtopic: It's so damn hard to read and understand the logic of this component with all these `in` <img src="https://media3.giphy.com/media/FcuiZUneg1YRAu1lH2/giphy.gif"/>

Reply:
i agree, currently refactoring this component in #563